pkg/registry/scheduling/rest/storage_scheduling.go

```diff
@@ -90,7 +90,8 @@ func AddSystemPriorityClasses() genericapiserver.PostStartHookFunc {
 				_, err := schedClientSet.PriorityClasses().Get(context.TODO(), pc.Name, metav1.GetOptions{})
 				if err != nil {
 					if apierrors.IsNotFound(err) {
-						_, err := schedClientSet.PriorityClasses().Create(context.TODO(), pc, metav1.CreateOptions{})
+						// create can mutate its input so we deep copy pc here since it is a global
```

Comment:
I think we can add a comment similar to this PR: https://github.com/kubernetes/kubernetes/pull/117107/files#diff-c2faa7dc1df07c6ec299bad68c04fb0b1623a0fd09219896bb919110aae8cbbdR121-R123. It’s the same root cause.